conans/client/generators/markdown.py

```diff
@@ -53,17 +53,21 @@
 
 generator_cmake_find_package_tpl = textwrap.dedent("""
     ### Generator ``cmake_find_package``
-    {% set cmake_find_package_filename = cpp_info.get_filename("cmake_find_package") %}
-    {% set cmake_find_package_name = cpp_info.get_name("cmake_find_package") %}
-
-    Generates the file Find{{ cmake_find_package_filename }}.cmake
+    Generates the file Find{{ cpp_info.get_filename("cmake_find_package") }}.cmake
```

Comment:
Why not use those jinja variables, and be calling ``cpp_info.xxxx`` several times?

Reply:
Yes, it makes sense now after the conflict, I gonna update it.

Reply:
Done